src/gen-stochastic.js

```diff
@@ -406,4 +406,92 @@ class MarkovChain {
 		return c;
 	}
 }
-exports.MarkovChain = MarkovChain;
```

Comment:
I think this is probably removed by accident? :)

Reply:
Haha indeed!